### PR TITLE
[onert/test] Revise benchmark scripts

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates benchmark scripts.
- Remove unused script: test_driver.sh
- Remove function: print_result_of_benchmark_test
- Rename script: benchmark_nnapi.sh -> benchmark_ops.sh
- Remove unused env variable setting
- Fix script args parsing bug

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Draft: https://github.com/Samsung/ONE/pull/10496
Related issue: https://github.com/Samsung/ONE/issues/9877

### CODE REVIEW

File: tests/scripts/benchmark_ops.sh [hunk 1]
```diff
@@ -55,6 +53,7 @@ do
             ;;
         --backends=*)
             BACKEND_LIST=${i#*=}
+            ;;
```
Comment (hseok-oh) [inline tests/scripts/benchmark_ops.sh:56]: Fix bug

File: tests/scripts/benchmark_ops.sh [hunk 2]
```diff
@@ -89,7 +88,7 @@ function run_benchmark_and_print()
     print_with_dots $MSG
     RESULT=$(get_result_of_benchmark_test $DRIVER_BIN $MODEL $LOG_FILE)
     echo "$RESULT ms"
-    print_result_of_benchmark_test "$MSG" "$RESULT" $RESULT_FILE
+    echo "$MSG $RESULT" > $RESULT_FILE
```
Comment (hseok-oh) [inline tests/scripts/benchmark_ops.sh:91]: `print_result_of_benchmark_test` is used here only

File: tests/scripts/benchmark_ops.sh [hunk 3]
```diff
@@ -124,7 +123,6 @@ function run_benchmark_test()
     local REPORT_MODEL_DIR=
 
     export COUNT=5
-    export ONERT_LOG_ENABLE=1
```
Comment (hseok-oh) [inline tests/scripts/benchmark_ops.sh:127]: Not used

File: tests/scripts/test_scheduler_with_profiling.sh [hunk 4]
```diff
@@ -185,7 +185,6 @@ function run_benchmark_test()
     unset BACKENDS
     echo "============================================"
     unset COUNT
-    unset USE_NNAPI
```
Comment (hseok-oh) [inline tests/scripts/test_scheduler_with_profiling.sh:188]: Not used

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**